pubsub.go

```diff
@@ -524,29 +524,29 @@ func (p *PubsubValueStore) handleNewMsgs(ctx context.Context, sub *pubsub.Subscr
 }
 
 func (p *PubsubValueStore) handleNewPeer(ctx context.Context, peerEvtHandler *pubsub.TopicEventHandler, key string) ([]byte, error) {
-	select {
-	case <-ctx.Done():
-		return nil, ctx.Err()
-	default:
-	}
-
-	var pid peer.ID
-
-	for {
+	tried := make(map[peer.ID]bool)
```

Comment:
Do we even want to track this? It may be better to just re-ask every peer when they re-join.

Reply:
Done.